ClipboardAPI/clipboard-change-event-explainer.md

```diff
@@ -200,7 +211,7 @@ Considered alternative - DataTransfer API: As per the current spec, the clipboar
 
 ### 7.1 Future permission prompting mechanisms
 
-In future, we may have additional ways of prompting the user for permissions - 1) By explicitly requesting for "clipboard-read" permission, the API for this is still under discussion (https://github.com/w3c/permissions/issues/158). 2) The user can be prompted for permissions as soon as the "addEventListener" method is called with "clipboardchange" in case the permissions are not already granted. This is still open for discussion as it is not a common pattern to prompt user for permissions when attaching event listeners. 3) The user can be prompted for permissions just before the browser dispatches the "clipboardchange" event. This way, the permissions prompt would appear only when required by the browser however web authors won't have control over when the prompt would be triggered which might not be desirable.  
+In future, we may have additional ways of prompting the user for permissions - 1) By explicitly requesting for "clipboard-read"/"clipboard-types-read" permission, the API for this is still under discussion (https://github.com/w3c/permissions/issues/158). 2) The user can be prompted for permissions just before the browser dispatches the "clipboardchange" event. This way, the permissions prompt would appear only when required by the browser however web authors won't have control over when the prompt would be triggered which might not be desirable.  
```

Comment:
When @ragoulik and I had chatted, we discussed potentially avoiding the `clipboard-types-read` permission from this proposal by simply never including custom formats in the `types` member of the ClipboardChange object. Did you all have a chance to discuss that further.